app/assets/javascripts/webchat.js.erb

```diff
@@ -25,6 +25,7 @@
         '/vat-registration/changes-to-your-details',
         '/government/organisations/hm-revenue-customs/contact/national-insurance-numbers',
         '/lost-national-insurance-number',
+        '/manage-your-tax-credits',
```

Comment:
JavaScript is the exception to this rule. Trailing commas break older IEs and are hard to track down.